hdfs/src/main/scala/akka/stream/alpakka/hdfs/scaladsl/HdfsSource.scala

```diff
@@ -70,7 +71,9 @@ object HdfsSource {
       }
       .takeWhile(_._1)
       .map(_._2)
-    Source.fromIterator(() => it)
+    Source
+      .fromIterator(() => it)
+      .addAttributes(Attributes(IODispatcher))
```

Comment:
Use `ActorAttributes.IODispatcher` here as well since the `DefaultAttributes.IODispatcher` is in the `impl` package.